api/src/paths/dwc/scrape-occurrences.ts

```diff
@@ -63,7 +63,22 @@ POST.apiDoc = {
   },
   responses: {
     200: {
-      description: 'Successfully scraped and uploaded occurrence information.'
+      description: 'Successfully scraped and uploaded occurrence information.',
+      content: {
+        'application/json': {
+          schema: {
+            type: 'object',
```

Comment:
does it make sense to make these required?

Reply:
yes